docker/api/Dockerfile

```diff
@@ -1,6 +1,6 @@
 ARG VERSION=unspecified
 
-FROM python:3.10.0-alpine
+FROM python:3.10.1
```

Comment:
Latest version of Python 3.10 and we should specify the underlying OS to avoid poor assumptions.

```suggestion
FROM 3.10.2-slim-bullseye
```